tests/phpunit/tests/taxonomy.php

```diff
@@ -33,6 +33,16 @@ public function test_get_post_taxonomy() {
 		}
 	}
 
+	function test_get_user_taxonomy() {
```

Comment:
This test seems unnecessary as 'user' has not built in taxonomies as 'post'.

Reply:
I wondered. On the original Trac ticket, someone asked whether this was right, but it never got answered.